src/wp-includes/default-filters.php

```diff
@@ -97,28 +106,29 @@
 
 // Format strings for display.
 foreach ( array( 'comment_author', 'term_name', 'link_name', 'link_description', 'link_notes', 'bloginfo', 'wp_title', 'widget_title' ) as $filter ) {
-	add_filter( $filter, 'wptexturize'   );
+	add_filter( $filter, 'wptexturize' );
 	add_filter( $filter, 'convert_chars' );
-	add_filter( $filter, 'esc_html'      );
+	add_filter( $filter, 'esc_html' );
 }
 
 // Format ClassicPress
-foreach ( array( 'the_content', 'the_title', 'wp_title' ) as $filter )
+foreach ( array( 'the_content', 'the_title', 'wp_title' ) as $filter ) {
```

Comment:
This array didn't get expanded either.

(I actually prefer it _not_ expanded.)